modules/local/drop_sample_annot.nf

```diff
@@ -1,5 +1,5 @@
-process DROP_ANNOTATION {
-    tag "DROP_annot"
+process DROP_SAMPLE_ANNOT {
+    tag "DROP_sample_annot"
     label 'process_low'
 
     conda "bioconda::drop=1.3.3"
```

Comment:
I would consider using another python container as I suspect that the drop biocontainer is quite big and is only used for this step.

Reply:
@jemten as we would need to download the container for the next step, wouldn't it be better to use this one than to download another container?

Reply:
Done